### DISCUSSION TITLE
java.lang.SecurityException: Permission Denial: package=flavor.packadge.name.test does not belong to uid=numbers

### DISCUSSION THREAD

**Discussion by ghost** [leancodepl/patrol#1479] [Q&A] [answered]
Howdy, guys! I'm trying to set up Patrol along with existing Flutter integration tests in the app with flavors. But build failing with:

```
dev/MainActivityTest.java:15: error: cannot find symbol
                instrumentation.setUp(MainActivity.class);
                                           ^
          symbol:   class MainActivity
          location: class MainActivityTest
        1 error
        
        FAILURE: Build failed with an exception.
```
Maybe someone faced the same error? Thanks in advance!

**UPD**: the problem was that I've created under "androidTest" folders according to the flavor's package name, but we have only one main AndroidManifest with a different package name.

**UPD 2**: now it throws "java.lang.SecurityException: Permission Denial: package=_flavor.packadge.name.**test**_ does not belong to uid=_numbers_" because of what I've described in UPD above.

**Comment by bartekpacia** [comment, marked as answer]
Hi @IvanHorchakov 👋🏻 

Glad that you've managed to solve the first problem 👍🏻 As for the second one, it looks similar to https://github.com/leancodepl/patrol/issues/1455.
